tslint.json

```diff
@@ -71,6 +71,10 @@
         "no-internal-module": true,
         "no-trailing-whitespace": true,
         "no-unsafe-finally": true,
+        "no-unused-variable": [
+          true,
+          "react"
+        ],
```

Comment:
To detect unused created or imported vars (except React when tslint detects any JSX element).